lib/safe-pg-migrations/plugins/blocking_activity_logger.rb

```diff
@@ -86,5 +86,10 @@ def log_blocking_queries(method)
 
       raise
     end
+
+    def format_start_time(start_time, reference_time = Time.now)
+      duration = (reference_time - start_time).round
```

Comment:
Is certain that the duration result will always be in seconds?

Reply:
Yes. In the worst case it will display `2800 seconds` which isn't too bad